libs/ui-toolkit/modals/src/lib/ui-modal/simple-modal/simple-modal.data.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Indicates interface for properties used in modal.
+ */
+export interface SimpleModalData {
+  bodyText?: string;
+  bodyHtml?: string;
+  bodyImage?: string;
+  socialMedia?: boolean;
+  socialMediaArray?: SocialMediaArray[];
+  noText?: string;
+  titleText: string;
+  yesText?: string;
+}
+
+/**
+ * Indicates specific interface for each button in modal.
```

Comment:
This needs some rewriting as well. The inferface indicates properties for each social media icon/link, not for the buttons of the modal